ducktape/cluster/remoteaccount.py

```diff
@@ -142,97 +317,182 @@ def kill_process(self, process_grep_str, clean_shutdown=True, allow_fail=False):
         for pid in pids:
             self.signal(pid, sig, allow_fail=allow_fail)
 
-    def scp_from_command(self, src, dest, recursive=False):
```

Comment:
We might reasonably remove the `scp_x_command` methods, but are we actually safe removing the `scp_from` and `scp_to` methods? Seems like these are used and are public API? Window of support & deprecation? Or are we just going to sed when we update the dependency version in kafkatest and muckrake?